src/cdk-experimental/testing/protractor/protractor-element.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {browser, ElementFinder} from 'protractor';
+import {TestElement} from '../test-element';
+
+/** A `TestElement` implementation for Protractor. */
+export class ProtractorElement implements TestElement {
+  constructor(readonly element: ElementFinder) {}
+
+  async blur(): Promise<void> {
+    return this.element['blur']();
+  }
+
+  async clear(): Promise<void> {
+    return this.element.clear();
+  }
+
+  async click(): Promise<void> {
+    return this.element.click();
+  }
+
+  async focus(): Promise<void> {
+    return this.element['focus']();
```

Comment:
Maybe add a comment here about why we need to access `focus` this way?

Reply:
I don't think this actually works. I just copied it in from google3 and intended to update it